### DISCUSSION TITLE
Question Add input Result

### DISCUSSION THREAD

**Discussion by hundredeuk2** [BoocampMonster/korean-relation-extraction#20] [Show and tell]
### Experiment status
> data:
>     train_path: data/train.csv
>     test_path: data/test_data.csv
>     entity_marker_mode: `dynamic`
>     entity_marker_num: `dynamic`
>     
> model:
>     model_name : klue/roberta-large
>     data_class : QueryDataset
>     model_class : PLMRNNModel # Automodel with biGRU 3 Layer Tanh
>     trainer: BaselineTrainer
>     saved_dir: query_entity_marker
>     loss : CEloss
>     metric_list:
>         metric1 : klue_re_micro_f1
>         metric2 : klue_re_auprc
>     optimizer : AdamW
>     dropout_rate : 0.2
> 
> train:
>     seed: 42
>     gpus: 1
>     batch_size: 16
>     max_epoch: 5
>     learning_rate: 1e-5
>     logging_step: 1
>     max_length : 270

![W B Chart 2022  11  28  오후 2_51_42](https://user-images.githubusercontent.com/73874591/204204159-ef86e737-3d37-470d-a534-f753765d8dae.png)

![W B Chart 2022  11  28  오후 2_51_52](https://user-images.githubusercontent.com/73874591/204204189-00703614-b7e1-4db9-8803-519e0f15516f.png)

**Comment by hundredeuk2** [comment]
@chanmuzi
